### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Connect to Local Server_Keyboard: Help texts and instructions for the input form fields are not accessible using keyboard.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#99]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106

**MAS ID Violated:** MAS2.1.1

**Repro Steps:**
1. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
2. Download and install "Microsoft IIS Administration" application.
3. Tab till "Skip This" button and Press Enter.
4. Verify that "Help" buttons that contains instructions and additional information present above the input fields are accessible using keyboard.

**Actual Result:**
"Help" buttons that contains instructions and additional information about the input element present above the input fields are not accessible using keyboard.

**Expected Result:**
Since these "Help" buttons are interactive controls and clicking on it using mouse displays instructions and additional information about the form field. So, same should be available for the keyboard only users.

**User Impact:**
Keyboard only users will be unable to know instructions and additional information about the form field.

**Note**: There is a "More information" link inside the tooltip popup that appears after selecting "Help" button. This link is also not accessible using the keyboard.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Attachment:**
[Bug 99_help buttons for input elements are not accessible using keyboard.pptx](https://github.com/Microsoft/IIS.WebManager/files/2323579/Bug.99_help.buttons.for.input.elements.are.not.accessible.using.keyboard.pptx)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Arjun553** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #C+E-H; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #MAS2.1.1; #Keyboard; #.NETWebsites; #Edge; #Chrome

**Comment by Roopesh-T** [comment]
**GitTags**: #A11ySev1;

**Comment by Yashi1201** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the www.iis.net home page will be removed.
